vllm/core/scheduler.py

```diff
@@ -255,10 +256,12 @@ def __init__(
         self,
         scheduler_config: SchedulerConfig,
         cache_config: CacheConfig,
+        parallel_config: ParallelConfig,
```

Comment:
can we make it backward compatible? e.g. add `parallel_config:Optional[ParallelConfig] = None`, and put it in the final arg.

Reply:
actually we just use the pp size, so it is safe to add `pp_size=1` stuff. We don't need the whole `ParallelConfig`.